### PR TITLE
cloud: add snappy restriction and split cmd

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Currently, this method supports importing one CSV file for one task into either
 
     2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.
 
-2. On the **Import** page, you can directly drag and drop your local file to the upload area, or click the upload area to select and upload the target local file. Note that you can upload only one CSV file of less than 50 MiB for one task.
+2. On the **Import** page, you can directly drag and drop your local file to the upload area, or click the upload area to select and upload the target local file. Note that you can upload only one CSV file of less than 50 MiB for one task. If the file is larger than 50 MiB, you can use the `split [-l line_count]` utility to split it into multiple smaller files (for Linux or macOS only). For example, run `split -l 100000 tidb-01.csv pages` to split a file named `tidb-01.csv` by line length `100000`, and the split files are named `pages<suffix>`.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-import-local-files.md:36]:
```suggestion
2. On the **Import** page, you can directly drag and drop your local file to the upload area, or click the upload area to select and upload the target local file. Note that you can upload only one CSV file of less than 50 MiB for one task. If the file is larger than 50 MiB, you can use the `split [-l ${line_count}]` utility to split it into multiple smaller files (for Linux or macOS only). For example, run `split -l 100000 tidb-01.csv pages` to split a file named `tidb-01.csv` by line length `100000`, and the split files are named `pages<suffix>`.
```

Comment (okJiang) [inline tidb-cloud/tidb-cloud-import-local-files.md:36]: can add some split steps here.

**Review by okJiang — COMMENTED** [review]
rest lgtm

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Currently, this method supports importing one CSV file for one task into either
 
     2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.
 
-2. On the **Import** page, you can directly drag and drop your local file to the upload area, or click the upload area to select and upload the target local file. Note that you can upload only one CSV file of less than 50 MiB for one task.
+2. On the **Import** page, you can directly drag and drop your local file to the upload area, or click the upload area to select and upload the target local file. Note that you can upload only one CSV file of less than 50 MiB for one task. If the file is larger than 50 MiB, you can use the `split [-l ${line_count}]` utility to split it into multiple smaller files (for Linux or macOS only). For example, run `split -l 100000 tidb-01.csv pages` to split a file named `tidb-01.csv` by line length `100000`, and the split files are named `pages${suffix}`.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-import-local-files.md:36]: 参考了 Snowflake 的 https://docs.snowflake.com/en/user-guide/data-load-considerations-prepare

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 3]
```diff
@@ -103,3 +103,23 @@ If you use `mysql` and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE fil
...
+file_path=$2
+file_extension="${file_path##*.}"
+file_name="${file_path%.*}"
+lines_per_file=$(( $(wc -l < $file_path) / $n ))
+split -d -a 1 -l $lines_per_file $file_path $file_name.
+for (( i=0; i<$n; i++ ))
+do
+    mv $file_name.$i $file_name.$i.$file_extension
+done
+```
```
Comment (okJiang) [inline tidb-cloud/tidb-cloud-import-local-files.md:125]: Give me a brief overview of this script. Users can input "n" and a file name, and then the script will divide the file into "n" equal parts while keeping the original file extension.

It is better to give users an example for that:

```
> sh ./split.sh 3 mytest.customer.csv
> ls -h | grep mytest
mytest.customer.0.csv
mytest.customer.1.csv
mytest.customer.2.csv
mytest.customer.csv
```

Comment (hfxsd) [reply tidb-cloud/tidb-cloud-import-local-files.md:125]: 好的 加好了

**Review by okJiang — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**